serve.js

```diff
@@ -47,22 +40,19 @@ if(!(args.hasOwnProperty('insecure') || args._.includes("insecure"))) {
         ca: ca
     };
 
-    app.all('*', ensureSecure); // at top of routing calls
-
-    http.createServer(app).listen(8090);
-    https.createServer(credentials, app).listen(4430, () => { //443
-        console.log('HTTPS Server running on port 4430'); //443
-    });
-
-    function ensureSecure(req, res, next){
-        if(req.secure){
+    app.all('*', (req, res, next) => {
+        if (req.secure) {
             // OK, continue
             return next();
         }
         // handle port numbers if you need non defaults
-        // res.redirect('https://' + req.host + req.url); // express 3.x
         res.redirect('https://' + req.hostname + req.url); // express 4.x
-    }
+    }); // at top of routing calls
+
+    http.createServer(app).listen(8090);
+    https.createServer(credentials, app).listen(4430, () =>//443
+        console.log('HTTPS Server running on port 4430') //443
+    );
```

Comment:
Remove outdated comments
```suggestion
    https.createServer(credentials, app).listen(4430, () =>
        console.log('HTTPS Server running on port 4430')
    );
```